wcwidth/wcwidth.py

```diff
@@ -154,35 +127,10 @@ def wcwidth(wc):
           Full-width (F) category as defined in Unicode Technical
           Report #11 have a column width of 2.
     """
-    # pylint: disable=C0103
-    #         Invalid argument name "wc"
-    ucs = ord(wc)
-
-    # NOTE: created by hand, there isn't anything identifiable other than
-    # general Cf category code to identify these, and some characters in Cf
-    # category code are of non-zero width.
-    if (0 == ucs or
-            0x034F == ucs or
-            0x200B <= ucs <= 0x200F or
-            0x2028 == ucs or
-            0x2029 == ucs or
-            0x202A <= ucs <= 0x202E or
-            0x2060 <= ucs <= 0x2063):
-        return 0
-
-    # C0/C1 control characters
-    if ucs < 32 or 0x07F <= ucs < 0x0A0:
-        return -1
-
-    # combining characters have indeterminate effects unless
-    # combined with additional characters.
-    if _bisearch(ucs, NONZERO_COMBINING):
-        return -1
-
-    return 1 + _bisearch(ucs, WIDE_EASTASIAN)
-
-
-def wcswidth(pwcs, n=None):
+    return wcswidth(wc, negative_returnval=True)
+
+
+def wcswidth(pwcs, n=None, negative_returnval=False):
```

Comment:
Could we document the need and purpose for ``netative_returnval`` keyword argument please? I see no use for it in your code, is this intentional? It requires tests before merge.